### DISCUSSION TITLE
F1 or Alt+H to show man page for current command

### DISCUSSION THREAD

**Discussion by YaLTeR** [nushell/nushell#8022] [Ideas]
Fish has a feature where pressing F1 or Alt+H will open a man page for the command you're currently trying to invoke. Closing the man page gets you straight back to what you were typing. E.g.:

```
$ ffmpeg -i input.mp4<F1>
=> opens ffmpeg man page
$ for file in *; echo $file; ffmpeg -i $file<F1> $file.mkv; end
=> opens ffmpeg man page because the cursor was in ffmpeg's arguments
$ echo -1 | sudo tee /proc/sys/kernel/perf_event_paranoid<F1>
=> opens tee man page (a few commands like sudo are hardcoded to open man page for their argument, because that's what you want)
$ echo -1 | sudo<F1> tee /proc/sys/kernel/perf_event_paranoid
=> opens sudo man page because the cursor is on sudo itself
```

This is a huge time saver, I use this all the time in fish. Would be great if nushell had a similar feature. I think the current F1 functionality could stay when the cursor is in the middle of a nushell command, or at the start of writing a new command.

**Comment by keinzirp** [comment]
I'm relatively new to nushell and It's been a few years since this has been proposed. It would be useful if I was able to see the help in the reedline while typing a command pipe as it cuts down on the amount that I need to look things up in a seperate help command. Often the recipes on the help tell me where I need to go next in the command chain. Not sure if I'm just missing something as I don't know this shell very well.
